examples/annotation_import/tiled.ipynb

```diff
@@ -20,12 +20,12 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    import gdal
</pre><p> </p><p>You aren't directly referencing this. I wouldn't import it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done </p>